reserver/reserver_param.py

```diff
@@ -21,4 +21,9 @@
     "url": r'^(http|https)://[a-zA-Z0-9.-_]+\.[a-zA-Z]{2,}(/\S*)?$',
 }
 INVALID_PACKAGE_PARAMETER_NAME_ERROR = "Given parameter doesn't exist among the supported user allowed parameters."
-INVALID_PACKAGE_PARAMETER_VALUE_ERROR = "Invalid value for {parameter} that should be a valid {regex}"
\ No newline at end of file
+INVALID_PACKAGE_PARAMETER_VALUE_ERROR = "Invalid value for {parameter} that should be a valid {regex}"
+INVALID_CONFIG_FILE_NAME_ERROR = "Given file name for user-defined setup.py params is not a string."
+PARAM_FILE_DOES_NOT_EXIST_ERROR = "Given file doesn't exist."
+INVALID_INPUT_USER_PARAM = "Invalid input for user params."
+UNEQUAL_PARAM_NAME_LENGTH_ERROR = "You should pass either one single file path to be used for setup.py parameters \
```

Comment:
Not sure about `setup.py` again.
Also, I prefer this:
```
`user_params_path` should be either a path or a list of paths to the config file(s).
```

Reply:
I update the error message, changing `setup.py` with `package`.